doc/Type/BagHash.pod6

```diff
@@ -143,6 +176,32 @@ say $a.sort;  # OUTPUT: «(2 => 2 3 => 1 4 => 1 18 => 1)␤»
 say $a.sort.reverse;  # OUTPUT: «(18 => 1 4 => 1 3 => 1 2 => 2)␤»
 =end code
 
+=head2 method add
+
+    method add(BagHash: \to-add, *%_ --> Nil)
+
+When given single item, C<add> adds it to the C<BagHash> or, if it was
+already present, increases its weight by 1.  When given a C<List>,
+C<Array>, C<Seq>, or any other type that C<does> the
+L<C«Iterator»|/type/Iterator> Role, C<add> adds each element of the
+C<Iterator> to the C<SetHash> or increments the weight of each element
+by 1.
+
+I<Note:> since version 2020.02.
+
+=head2 method remove
+
+    method remove(BagHash: \to-remove, *%_ --> Nil)
+
+When given single item, C<remove> reduces the weight of that item by
```

Comment:
I still don't like repetition of text, because if there's an error, that error is repeated. At least you're not saying "remove removes" :-)
Alternative: "when the argument C<to-remove> is the name of a single item"... You also need to be precise about what you mean by "a single item". Does it stringify, for instance? Can you use objects? Junctions?